viewer/packages/360-images/src/collection/Image360Collection.ts

```diff
@@ -115,6 +115,42 @@ export interface Image360Collection {
    */
   setIconsVisibility(visible: boolean): void;
 
+  /**
+   * Check weather the hidden icons are visible
+   * @returns true is hidden icons are visible
+   */
+  isHiddenIconsVisible(): boolean;
+
+  /**
+   * Set the hidden icons visible
+   * @param visible
+   */
+  setHiddenIconsVisible(visible: boolean): void;
+
+  /**
+   * Get the opacity of the images
+   * @returns The opacity of the images
```

Comment:
I think I'd prefer the term `OccludedIcon` rather than `HiddenIcon`. (Hidden could mean something else)